### DISCUSSION TITLE
Lualine weird separators

### DISCUSSION THREAD

**Discussion by MVP-Harry** [nvim-lualine/lualine.nvim#536] [Configuration]
Hi,

I've just installed Lualine, and it looks great! However, the separators look weird: 
![image](https://user-images.githubusercontent.com/64049111/148485053-8140f248-89c4-472a-b9a6-4d746f68fb9c.png)
I think the issue probably comes from this two lines of code:
```
    component_separators = { left = '|', right = ''},
    section_separators = { left = '', right = ''},
``` 

I've done some research, and I think it might be because I'm missing some fonts. However, I have already installed powerline fonts, and can't figure out what's causing the problem. 

Thanks!

**Comment by MVP-Harry** [comment]
A little update: turns out that powerline font does fix some issue. However, there is some new weird characters: 
![image](https://user-images.githubusercontent.com/64049111/148486713-f9759044-26b8-4ead-9b61-6a7871a1bf0a.png)

I have already disabled the icons, so I'm not sure what's causing the trouble.

**Comment by pocco81** [comment]
Hey @MVP-Harry! There are a couple of factors the affect the display of characters, not only on lualine but also in your terminal:

#### The terminals' font rendering algorithm:

Terminals' do it differently, hence while Alacritty users may see this:

![image](https://user-images.githubusercontent.com/58336662/148659357-2e6793ff-47cd-4f35-a142-ea5318ca4911.png)

This is what they'd see on Kitty:

![image](https://user-images.githubusercontent.com/58336662/148659375-50838df8-c128-45b0-b0cf-edd2b742f0af.png)

#### The font

There are some fonts that have "_broken_" icons. For example:

![image](https://user-images.githubusercontent.com/58336662/148659528-4566e889-8041-457b-85fb-8db35d463781.png)

As you can see here both the `->` and the `==` which are supposed to be displayed as ligatures get displayed weirdly.


### Solution:

I'd recommend you to:

1. Try a different font (personally recommend Nerd Fonts, my favorite one is [Iosevka](https://github.com/ryanoasis/nerd-fonts/tree/master/patched-fonts/Iosevka))
2. Try a different terminal (personal favorite is [Kitty](https://github.com/kovidgoyal/kitty))
3. Play with the font size, that also affects this.

Hope it helps!

**Reply by johnsonhong997** [reply]
The example looks so great. I use catppuccin theme too. Could you share your configuration of lualine?
